### PR TITLE
Update seeding-your-database.mdx 

### PR DESCRIPTION
**Pull request by p5quared**
## What kind of change does this PR introduce?

**Documentation Update**

## Additional Context

Updates to docs on seeding database with Snaplet.

Also see
[supabase/.../seeding-your-database](https://supabase.com/docs/guides/cli/seeding-your-database)

[snaplet/.../seed](https://docs.snaplet.dev/getting-started/quick-start/seed)

## Issue
#21089

### CODE REVIEW

**Comment by charislam** [comment]
@peterp, do you want to look this one over?

**Comment by khaya-zulu** [comment]
> @peterp, do you want to look this one over?

hi @charislam taking a look now... thanks @p5quared for opening this up.

File: apps/docs/pages/guides/cli/seeding-your-database.mdx
Comment (khaya-zulu) [inline apps/docs/pages/guides/cli/seeding-your-database.mdx:85]: small change.

```suggestion
    email: (ctx) =>
      copycat.email(ctx.seed, {
        domain: 'acme.org',
      })
```

File: apps/docs/pages/guides/cli/seeding-your-database.mdx
Comment (khaya-zulu) [inline apps/docs/pages/guides/cli/seeding-your-database.mdx:71]: Just looking at the schema [here](https://github.com/supabase/supabase/blob/cb70067743cd3d414fea9b5f6658b03597c1dc3a/apps/docs/public/img/guides/cli/snaplet-example-schema.png). I think its okay to keep it as `createdBy`, that way we don't have to change the image?

File: apps/docs/pages/guides/cli/seeding-your-database.mdx
Comment (khaya-zulu) [inline apps/docs/pages/guides/cli/seeding-your-database.mdx:82]: Then same here, instead of `author`, let's use `userCreatedBy`

**Review by khaya-zulu — CHANGES_REQUESTED** [review]
Just a few small comments, but thank you again @p5quared for pick this up for us 🙂 Let me know when you ready for another review.

File: apps/docs/pages/guides/cli/seeding-your-database.mdx
Comment (p5quared) [reply apps/docs/pages/guides/cli/seeding-your-database.mdx:71]: Ah shoot I actually thought that and then somehow went ahead and changed them anyways; thanks!

**Review by khaya-zulu — APPROVED** [review]
Approved! thanks once again @p5quared, hope the swag gets to you 😄 I'll let @peterp know, you might have find us in our discord.

File: apps/docs/pages/guides/cli/seeding-your-database.mdx
Comment (charislam) [inline apps/docs/pages/guides/cli/seeding-your-database.mdx:64]:
```suggestion
Suppose you have a database with the following schema:
```

File: apps/docs/pages/guides/cli/seeding-your-database.mdx
Comment (charislam) [inline apps/docs/pages/guides/cli/seeding-your-database.mdx:68]:
```suggestion
You can use the Snaplet configuration file `seed.mts` to define the values you want to generate. For example
```

File: apps/docs/pages/guides/cli/seeding-your-database.mdx
Comment (charislam) [inline apps/docs/pages/guides/cli/seeding-your-database.mdx:92]:
```suggestion
Running `npx tsx seed.mts > supabase/seed.sql` generates the relevant SQL statements inside your `supabase/seed.sql` file:
```

**Review by charislam — APPROVED** [review]

**Pull request merged**
